Jenkinsfile

```
@@ -17,6 +17,12 @@ pipeline {
                 ])
             }
         }
+        
+        stage ('Check if dist file were built') {
```

Nice @tockards, this would be very useful. Perhaps fail early instead of echoing. We should mob it!